lib/services/map_helper.dart

```diff
@@ -14,8 +16,33 @@ class MapHelper {
       LatLng(45.49741, -73.57819),
       LatLng(45.49781, -73.57906)
     ];
+    return (latLng == null
+        ? false
+        : boundsFromLatLngList(coordsList).contains(latLng));
+  }
 
-    return boundsFromLatLngList(coordsList).contains(latLng);
+  static bool isWithinSGW(LatLng latLng) {
```

Comment:
- isWithinSGW is completely unused except for testing, should this be part of the conditions in the map_notifier.dart class?

Reply:
yes this has now been removed, good catch